bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZpToTpTp/ZpToTpTp_conf/Zt_madspin_card.dat

```diff
@@ -0,0 +1,11 @@
+set ms_dir ./madspingrid
+set max_running_process 1
+set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+#set BW_cut 15                # cut on how far the particle can be off-shell
+set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+
+
+# specify the decay for tops
+decay t > w+ b
+decay t~ > w- b~
```

Comment:
Hi, the strategy is to decay W and Z with Pythia?

Reply:
Hi, yes that was the idea because it seemed easier to me, but I can decay the W and Z here with madspin as well if it is preferable

Reply:
Well it's up to you so it's fine for me

Reply:
Just to be clear, for our analysis (an anomaly search), either is fine. So I have now changed to doing the decays in madspi